python/nammu/controller/NammuController.py

```diff
@@ -44,7 +44,7 @@
 from javax.swing.filechooser import FileNameExtensionFilter
 from javax.swing.text import DefaultCaret
 from pyoracc.atf.common.atffile import AtfFile
-from requests.exceptions import RequestException
+from requests.exceptions import RequestException, ConnectTimeout
```

Comment:
Why is ConnectTimeout needed? This might be leftover from trying other things below.